schemas/control-statement-schema.json

```diff
@@ -0,0 +1,291 @@
+      "items": {
+        "oneOf": [
+          {
+            "type": [
+              "string",
+              "null"
+            ],
+            "minLength": 2
+          },
+          {
+            "$ref": "#/definitions/control_mechanism"
+          }
+        ]
+      }
+    },
+    "no_owners_reasons": {
+      "description": "The reason why no controlling entities are declared, for example because the entity has been unable to discover them, or there are no controlling entities (e.g. 100 people owned 1% each), or the companies is exempt from reporting (as some listed companies are in some jurisdictions)",
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
+    "uid": {
+      "description": "Unique id/reference for the control relationship entity record if the external source has given it one",
+      "type": "string"
+    },
+    "url": {
+      "description": "Persistent URL on the external source for the control relationship entity record, if there is one",
+      "type": "string"
+    },
+    "source_url": {
+      "description": "Source url of the web page for the filing or if there is not persistent URL the page from which it can be found (e.g. search page). This field is required.",
+      "type": "string",
+      "minLength": 11
+    },
+    "confidence": {
+      "description": "Confidence in accuracy of data",
+      "type": "string",
+      "enum": [
+        "HIGH",
+        "MEDIUM",
+        "LOW"
+      ]
+    },
+    "start_date": {
+      "description": "Date on which tcontrol started",
+      "type": "string",
+      "format": "date"
+    },
+    "end_date": {
+      "description": "Date on which control ended",
+      "type": "string",
+      "format": "date"
+    },
+    "retrieved_at": {
+      "description": "Date-time this was retrieved from the source",
+      "type": "string",
+      "format": "date-time"
+    },
+    "sample_date": {
+      "description": "Date on which we know this to be true (for example the date the control statement was published)",
+      "type": "string",
+      "format": "date"
+    },
+    "other_attributes": {
+      "description": "Use for other attributes for which we don't yet have curated schema attributes",
+      "type": "object"
+    }
+  },
+  "additionalProperties": false,
+  "required": [
+    "control_level",
+    "subject_entity",
+    "retrieved_at",
+    "source_url",
+    "sample_date"
+  ],
+  "definitions": {
+    "control_mechanism": {
+      "title": "Control Mechanism",
+      "description": "A specific mechanism(s) by which controls of a company is exercised",
+      "oneOf": [
+        {
+          "properties": {
+            "mechanism_type": {
+              "enum": [
+                "share_ownership"
+              ]
+            },
+            "mechanism_properties": {
+              "properties": {
+                "number_of_shares": {
+                  "description": "Number of shares, if known",
```

Comment:
Do we need to have `if known` everywhere? If the field isn't required then it should be obvious enough? Put it another way: we don't have `if known` on all the other optional fields...